addOns/automation/src/main/java/org/zaproxy/addon/automation/ExtensionAutomation.java

```diff
@@ -650,6 +643,69 @@ public void execute(CommandLineArgument[] args) {
         }
     }
 
+    private void runPlanCommandLine(String source) {
+        URI uri = createtUri(source);
+        if (uri != null) {
+            Path file;
+            try {
+                HttpMessage message = new HttpMessage(uri);
+                new HttpSender(HttpSender.MANUAL_REQUEST_INITIATOR).sendAndReceive(message);
```

Comment:
Is it worth checking the response content type? And add timeouts / size limits? I don't think we're doing that for the API import add-ons, but we probably should (doesn't need to be in this PR).

Reply:
We validate if YAML when parsing it. For the other measures I thought on that and agree that we should do that at some point (even if the user wants to shoot itself it will be able to no matter what).